src/checks/components/NewDeadmanCheckEO.tsx

```diff
@@ -39,7 +39,9 @@ const NewCheckOverlay: FunctionComponent = () => {
         view,
       })
     )
-    return () => dispatch(resetAlertBuilder)
+    return () => {
```

Comment:
This expects to return `void`, but due to the implicit return of fat arrow functions, it was returning the return value of the `dispatch` call